wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

```diff
@@ -1,123 +1,113 @@
 
-### Divisor 1/2
+The 1/1 snap divisor allows for hit objects to be placed on every *full beats*. It is being represented by white-coloured ticks on the editor timeline.
 
-![Divisor 1/2](/wiki/shared/BSD_1_2.jpg "Divisor 1/2")
+Due to its simplicity and ease of use, this snap divisor is most commonly used while mapping Easy difficulties.
 
-- Half Beats
-- Adds red ticks
+## 1/2 snap divisor
 
-### Divisor 1/3
+![1/2 snap divisor in the osu!/osu!taiko/osu!catch editor](/wiki/shared/BSD_1_2.jpg "1/2 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-![Divisor 1/3](/wiki/shared/BSD_1_3.jpg "Divisor 1/3")
+![1/2 snap divisor in the osu!mania editor](/wiki/shared/1_2_m.jpg "1/2 snap divisor in the osu!mania editor")
 
-- Triplets*
-- Has two purple ticks between each white tick
-- Compound timing
+The 1/2 snap divisor allows for hit objects to be placed on every *half beats* and its multiples (e.g. 1/2). It is being represented by red-coloured ticks on the editor timeline.
 
-### Divisor 1/4
+## 1/3 snap divisor
 
-![Divisor 1/4](/wiki/shared/BSD_1_4.jpg "Divisor 1/4")
+![1/3 snap divisor in the osu!/osu!taiko/osu!catch editor](/wiki/shared/BSD_1_3.jpg "1/3 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-- Quarter Beats
-- Adds blue ticks, in addition to the red ticks
-- Useful for general mapping
+![1/3 snap divisor in the osu!mania editor](/wiki/shared/1_3_m.jpg "1/3 snap divisor in the osu!mania editor")
 
-### Divisor 1/6
+The 1/3 snap divisor allows for hit objects to be placed on *triplets*, which is the third fraction of a beat and its multiples (e.g. 1/3 and 2/3). It is being represented by purple-coloured ticks on the editor timeline.
 
-![Divisor 1/6](/wiki/shared/BSD_1_6.jpg "Divisor 1/6")
+Although relatively rare, there are some songs and music genres out there (e.g. swing jazz) that are primarily utilizing this snap divisor throughout.
 
-- Double Triplets*
-- Has two purple ticks between each red tick
-- Compound timing
+## 1/4 snap divisor
 
-### Divisor 1/8
+![1/4 snap divisor in the osu!/osu!taiko/osu!catch editor](/wiki/shared/BSD_1_4.jpg "1/4 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-![Divisor 1/8](/wiki/shared/BSD_1_8.jpg "Divisor 1/8")
+![1/4 snap divisor in the osu!mania editor](/wiki/shared/1_4_m.jpg "1/4 snap divisor in the osu!mania editor")
 
-- Eighth Beats
-- Adds yellow ticks, in addition to the red and blue
+The 1/4 snap divisor allows for hit objects to be placed on every *quarter beats*, which is the fourth fraction of a beat and its multiples (e.g. 1/4 and 3/4). It is being represented by blue-coloured ticks on the editor timeline.
 
-The 1/3 and 1/6 are special because it means your BPM is divisible by three, but this is *rare*.
+Due to its versatility, this snap divisor is extremely popular among mappers and is frequently used in all sorts of maps in general. It is also the default setting that is being designated by the editor upon importing a new audio file.
```

Comment:
```suggestion
Due to its versatility, this snap divisor is extremely popular among mappers and is frequently used in all sorts of beatmaps in general. It is also the default setting that is designated by the editor upon importing a new audio file.
```
not sure if this line is necessary. either way, line should be rewritten to use neutral tone (this goes for all other lines like this)